assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -109,7 +128,7 @@ function Carousel() {
 		<Wrapper>
 			<Area area="left-navigation">
 				<LeftArrow
-					isHidden={ ! hasHorizontalOverflow }
+					isHidden={ ! hasHorizontalOverflow || 0 === scrollPercentage }
```

Comment:
Would be great to add a constant with explanatory name for `0 === scrollPercantage` for readability (and below, too)

Reply:
Done in 19f36d2ee0e838ea86a1c52d56928d9a951a3040